agent/pkg/providers/stats_provider.go

```diff
@@ -45,9 +45,15 @@ type AccumulativeStatsProtocol struct {
 	Methods []*AccumulativeStatsCounter `json:"methods"`
 }
 
+type AccumulativeStatsProtocolTime struct {
+	ProtocolsData []*AccumulativeStatsProtocol `json:"protocols"`
+	Time          int64                        `json:"timestamp"`
+}
+
 var (
-	generalStats = GeneralStats{}
-	bucketsStats = BucketStats{}
+	generalStats            = GeneralStats{}
+	bucketsStats            = BucketStats{}
+	internalBucketThreshold = time.Minute * 1
```

Comment:
can it be const?

Reply:
Sure, changed in this https://github.com/up9inc/mizu/pull/1157/commits/5909395b053adc932ee67e2c4547b98bc77962a8